File: Others/easy/ClassPhotos.py
def classPhotos(redShirtHeights, blueShirtHeights):

    redShirtHeights.sort()  # we consider this as backrow
    blueShirtHeights.sort()  # we consider this as frontrow

    if redShirtHeights[0] == blueShirtHeights[0]:
        return False

    if redShirtHeights[0] < blueShirtHeights[0]:
        redShirtHeights, blueShirtHeights = blueShirtHeights, redShirtHeights

    for i in range(len(redShirtHeights)):
        if redShirtHeights[i] <= blueShirtHeights[i]:
            return False

    return True

File: Golang/easy/class-photos.go
package main

import "sort"

func cmpArrays(arr1 []int, arr2 []int) bool {
	for idx := range arr1 {
		if arr1[idx] <= arr2[idx] {
			return false
		}
	}
	return true
}

func ClassPhotos(redShirtHeights []int, blueShirtHeights []int) bool {
	sort.Ints(redShirtHeights)
	sort.Ints(blueShirtHeights)

	if redShirtHeights[0] > blueShirtHeights[0] {
		return cmpArrays(redShirtHeights, blueShirtHeights)
	} else if redShirtHeights[0] < blueShirtHeights[0] {
		return cmpArrays(blueShirtHeights, redShirtHeights)
	}

	return false
}
